TaskManager/Modules/Home/View/HomeViewController.swift

```diff
@@ -7,13 +7,22 @@
 
 import UIKit
 
-enum Section: String, CaseIterable {
+enum Section: String {
     case Active
     case Completed
+    
+    func localizedString() -> String {
+        return NSLocalizedString(self.rawValue, comment: "")
```

Comment:
lets create constants with structure similar to
enum Localized {
enum Home {
    static let editButton = localizedString("home.editButton")
 }
}